### PR TITLE
re-organise support page for UniversalProfile.cloud

### PR DESCRIPTION
**Pull request by CJ42**
image

### CODE REVIEW

**Review by mustafademiray — APPROVED** [review]

File: docs/universalprofilecloud/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+!(View profile button dashboard)[/universalprofilecloud/view-profile-button-dashboard.png]
+
+## Manage my Universal Profile
+
+You can connect your Universal Profile to [my.universalprofile.cloud](https://my.universalprofile.cloud/) to manage it. From the dashboard, you can:
+
+- See the Universal Profiles you have created.
+- Add Recovery with 2FA authentication.
+- Recover a Universal Profile you have created.
+- See you relayer quota
```
Comment (JeneaVranceanu) [inline docs/universalprofilecloud/dashboard.md:40]:
```suggestion
- See your quota left on relayer
```

File: docs/general/faq.md [hunk 2]
```diff
@@ -30,4 +30,16 @@ If your deployed assets do not appear on your profile on [universalprofile.cloud
...
+
+Unlike regular wallets that are Externally Owned Accounts (EOA), a Universal Profile is an account powered by a **smart contract**.
+
+With EOA wallets, _transactions are made directly by the address itself_ (which own the funds like native tokens, ERC20 tokens or NFTs).
+
+With a Universal Profile, transactions are _executed either via a relayer or directly by one of it's controllers_.
+
+When you use your Universal Profile to transfer LYX or any assets (tokens or NFTs), some custom logic must be executed before transferring the asset itself (_this includes checking the permissions, if the UP has enough LYX that it is trying to send, EVM internal logic, etc..._). Therefore the actual _"transfer"_ is part of a larger execution logic. This is the reason why actual LYX and token transfers appear as internal transactions.
+
+In a transaction on the block explorer, you can see the actual transfer being made as part of the **internal transactions**. For example in the screenshot below, you can see the LYX transfer being one of the internal transactions (the other internal transactions before and after are related to what is mentioned above).
```
Comment (JeneaVranceanu) [inline docs/general/faq.md:45]: `For example in the screenshot below` - there's no screenshot below.

File: docs/universalprofilecloud/faq.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+description: General questions related to universalprofile.cloud
+---
+
+# FAQ
+
+## General
+
+### Where can I see my Universal Profile?
+
+You can find your Universal Profile on the Universal Profile Explorer at [universalprofile.cloud](https://universalprofile.cloud/). Just select the correct network on the bottom left corner and use the global search bar to search for your name.
```
Comment (JeneaVranceanu) [inline docs/universalprofilecloud/faq.md:13]: We are searching `for a profile` and searching `by a name` (or address):
```suggestion
You can find your Universal Profile on the Universal Profile Explorer at [universalprofile.cloud](https://universalprofile.cloud/). Just select the correct network on the bottom left corner and use the global search bar to search by your name.
```

File: docs/universalprofilecloud/wallet.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+---
+sidebar_label: '🪙 Wallet & Assets'
+sidebar_position: 2
+---
+
+# Wallet & Assets
+
+import React from 'react'
+import LoomVideo from '@site/src/components/LoomVideo'
```
Comment (JeneaVranceanu) [inline docs/universalprofilecloud/wallet.md:9]:
```suggestion
```

File: docs/universalprofilecloud/wallet.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+## Transfer Tokens & NFTs from my Univeral Profile
+
+The Universal Profile Extension **does not enable you to transfer tokens directly from the extension itself**.
+
+Instead, it is designed to work with various decentralized applications (dApps) that can seamlessly integrate wallet features in their UI. T
+
+with your Universal Profile and interact with its information and controllers. You can connect your Universal Profile to [wallet.universalprofile.cloud](https://wallet.universalprofile.cloud/) to send token transactions.
```
Comment (JeneaVranceanu) [inline docs/universalprofilecloud/wallet.md:17]: Something is missing here:
```suggestion
Instead, it is designed to work with various decentralized applications (dApps) that can seamlessly integrate wallet features in their UI. T ????
???
??? with your Universal Profile and interact with its information and controllers. You can connect your Universal Profile to [wallet.universalprofile.cloud](https://wallet.universalprofile.cloud/) to send token transactions.
```

File: docs/universalprofilecloud/wallet.md (hunk 4 above)
Comment (CJ42) [reply docs/universalprofilecloud/wallet.md:9]: Yes there are Loom videos embedded in this page

**Pull request merged**